compiler/src/iree/compiler/Codegen/LLVMCPU/ExpandF16MaxFToF32Pass.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+#include "iree/compiler/Codegen/LLVMCPU/LLVMCPUPasses.h"
+#include "iree/compiler/Codegen/PassDetail.h"
+#include "iree/compiler/Codegen/Transforms/Transforms.h"
+#include "iree/compiler/Codegen/Utils/MarkerUtils.h"
+#include "iree/compiler/Codegen/Utils/Utils.h"
+#include "llvm/Support/Debug.h"
+#include "mlir/Dialect/Vector/Transforms/LoweringPatterns.h"
+#include "mlir/Dialect/Vector/Transforms/VectorRewritePatterns.h"
+#include "mlir/Dialect/Vector/Transforms/VectorTransforms.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include "mlir/Dialect/Linalg/IR/Linalg.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/Dialect/Arith/IR/Arith.h"
+
+namespace mlir {
+namespace iree_compiler {
+
+namespace {
+
+struct ExpandF16MaxFToF32Pattern : public OpRewritePattern<arith::MaxFOp> {
+  public:
+    using OpRewritePattern<arith::MaxFOp>::OpRewritePattern;
+
+    LogicalResult matchAndRewrite(arith::MaxFOp op, 
+                                  PatternRewriter &rewriter) const override{
+      Type resultType = op.getLhs().getType();
+      llvm::outs()<<"\n\n Added \n\n";
```

Comment:
Please wrap these within

```
LLVM_DEBUG({ ...

})
```

you should be able to find examples of this in the code base. Also the convention is that you write to `llvm:dbgs()`... instead of `llvm::outs`. Such debug prints should be within `LLVM_DEBUG({..})` so that it is not on by default but can be enabled using flag on debug builds for debugging help.

Reply:
Understood 
Thank you